src/data/lovelace_custom_cards.ts

```diff
@@ -6,19 +8,37 @@ export interface CustomCardEntry {
   documentationURL?: string;
 }
 
+export interface CustomTileFeatureEntry {
+  type: string;
+  name?: string;
+  supported?: (stateObj: HassEntity) => boolean;
+  configurable?: boolean;
+}
+
 export interface CustomCardsWindow {
   customCards?: CustomCardEntry[];
+  customTileFeatures?: CustomTileFeatureEntry[];
 }
 
 export const CUSTOM_TYPE_PREFIX = "custom:";
 
+export const isCustomType = (type: string) =>
```

Comment:
https://github.com/home-assistant/frontend/blob/dev/src/common/structs/is-custom-type.ts#L3